app-text/zathura/zathura-0.5.2-r5.ebuild

```diff
@@ -58,11 +58,11 @@ src_configure() {
 	meson_src_configure
 }
 
+src_test() {
+	virtx meson_src_test
+}
+
 src_install() {
 	meson_src_install
 	doman "${WORKDIR}"/man/zathura*
```

Comment:
If we're going the live template route (and I hope we can), we should put this in the live ebuild too but guard it with if [[ ${PV} != 9999 ]] ; then
	doman ...
fi

Reply:
would you be fine with `[[ ${PV} != 9999 ]] && doman ..`?

Reply:
wfm